File: DKS-SPA/src/app/core/_models/sample-track-report-dto.ts
import { ModelInterface } from "./interface/model-interface";

export class SampleTrackReportDto implements ModelInterface{
     sampleNo :string;
     article :string;
     modelNo :string;
     modelName :string;
     team :string;

     stage :string;
     test :string;
     barCodeNo :string;
     devTransDate :Date;
     devTransSn :string;

     devKeeper :string;
     devTransferer :string;
     overDays: number;
}

File: DKS-API/DTOs/SampleTrackReportDto.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class SampleTrackReportDto
    {

        public string SampleNo { get; set; }
        public string Article { get; set; }
        public string ModelNo { get; set; }
        public string ModelName { get; set; }
        public string Team { get; set; }

        public string Stage { get; set; }
        public string Test { get; set; }
        public string BarCodeNo { get; set; }
        public DateTime DevTransDate { get; set; }
        public string DevTransSn { get; set; }

        public string DevKeeper { get; set; }
        public string DevTransferer { get; set; }
        public int OverDays { get; set; }
        public string DeptName { get; set; }
        public string TransferName { get; set; }

        

    }
}